### DISCUSSION TITLE
How to sort actions list?

### DISCUSSION THREAD

**Discussion by jhl0214** [community/community#143016] [Actions] [answered]
### Select Topic Area

Question

### Body

Hi, in the actions tab, the list is sorted in descending order of the create time. However, if I search with workflow name, they are not sorted at all. Is there a way to keep them sorted in descending order?

**Comment by TzivyaFireman** [comment, marked as answer]
As far as I know, it is not possible to filter or sort search results by creation time directly within the GitHub Actions UI when searching by workflow name. 


Alternatively, you can use the GitHub API to perform a custom query that returns filtered and sorted results based on creation time.

**Reply by jhl0214** [reply]
Can the API also filter by substring of a workflow run name?

**Reply by TzivyaFireman** [reply]
From what I understand, the GitHub API doesn’t support filtering by substring in a workflow run name.
It seems you’d need to fetch all runs and filter by the substring in your code.

**Reply by jhl0214** [reply]
Got it. Can you please give the link for the API which will let me sort by the create time? Thanks in advance!

**Reply by TzivyaFireman** [reply]
You can use this API link to get the workflow runs for your repository:
https://api.github.com/repos/YOUR_USERNAME/YOUR_REPO/actions/runs

Note: The API doesn’t support sorting by creation time; you’ll need to sort the results yourself.

**Reply by jhl0214** [reply]
Thank you. I actually won't be able to use api as I have tons of workflow jobs, so unless I can filter by job name, there will be too many records I don't need... but that you for the info!
